bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/binding/ThingHandler.java

```diff
@@ -174,4 +175,24 @@
      */
     void handleRemoval();
 
+    /**
+     * This method should return a unique identifier of the device handled by this {@link ThingHandler}.
```

Comment:
Ok, understood. Would you then opt for implementing the getRepresentationProperty in the thing itself, instead of the handler? And like in DiscoveryResult I assume you do not want me to add a getRepresentaionPropertyValue() convenience method here?